lighthouse-cli/bin.js

```diff
@@ -5,51 +5,57 @@
  */
 'use strict';
 
-import {existsSync} from 'fs';
-import * as path from 'path';
+const existsSync = require('fs').existsSync;
+const path = require('path');
 
-import * as Commands from './commands/commands';
-import * as Printer from './printer';
-import {getFlags} from './cli-flags';
-import {runLighthouse} from './run';
+const commands = require('./commands/commands.js');
+const printer = require('./printer.js');
+const getFlags = require('./cli-flags.js').getFlags;
+const runLighthouse = require('./run').runLighthouse;
 
 const log = require('lighthouse-logger');
+// @ts-ignore
 const perfOnlyConfig = require('../lighthouse-core/config/perf.json');
+// @ts-ignore
 const pkg = require('../package.json');
 const Sentry = require('../lighthouse-core/lib/sentry');
 
-// accept noop modules for these, so the real dependency is optional.
-import {updateNotifier} from './shim-modules';
-import {askPermission} from './sentry-prompt';
+const updateNotifier = require('update-notifier');
```

Comment:
> intentionally not shimming anymore?

yeah, I think(?) it only added value to users not installing from `package.json` and managing all dependencies themselves (since we didn't add them as `peerDependencies`), and I don't think we have any of those users anymore, so it seemed like a good time to remove. No big deal to add back if we want to keep though

Reply:
yeah that seems fine, better to be clear about our non users than half support :)